001/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	slice := []interface{}{1, "2", 10, "11"}
+	for _, v := range slice {
+		switch v := v.(type) {
+		case int:
+			fmt.Println(fmt.Sprintf("%02d", v))
+		case string:
+			i, _ := strconv.Atoi(v)
+			fmt.Println(fmt.Sprintf("%02d", i))
+		}
+	}
```

Comment:
`%v`を使えば各データ型のdefault formatで出力されるので、
今回のケースでは型で分岐なしに記述できます。

```suggestion
	for _, v := range slice {
			fmt.Println(fmt.Sprintf("%02v", v))
	}
```

https://pkg.go.dev/fmt#hdr-Printing